### ISSUE TITLE
Allow disable pr body as commit message when squash merge

### ISSUE THREAD

**Issue by viceice** [go-gitea/gitea#15914]
- Gitea version (or commit ref): 1.14.2
- Git version: official gitea docker image from above
- Operating system: official gitea docker (kubernetes helm)
  
  
  
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
- Log gist:

## Description

PR #13071 added the pr body to the squash commit message. I like to have an option to disable this behavior.
I'm using [renovate](/renovatebot/renovate)  to manage my dependencies and always squash my PR's.
The renovate PR descriptions can be very long because of changelogs, so i would like to exclude the content as i can do for commits with:

```ini
[repository.pull-request]
DEFAULT_MERGE_MESSAGE_COMMITS_LIMIT=0
```

I would be happy to help to add such an option if it is welcome. 🙃 

## Screenshots

![image](https://user-images.githubusercontent.com/1798109/118615033-9bc64c00-b7c0-11eb-9904-4f69ea8bf4a8.png)

**Comment by viceice** [comment]
I think it should repect `DEFAULT_MERGE_MESSAGE_SIZE=0` here:
https://github.com/go-gitea/gitea/blob/1da0d156bc335d01f6ce99aa41d2b8d9677bdbbc/services/pull/pull.go#L573-L577

**Comment by viceice** [comment]
Any feedback?

**Comment by zeripath** [comment]
Set POPULATE_SQUASH_COMMENT_WITH_COMMIT_MESSAGES = true and DEFAULT_MERGE_MESSAGE_SIZE=0

**Comment by viceice** [comment]
> Set POPULATE_SQUASH_COMMENT_WITH_COMMIT_MESSAGES = true and DEFAULT_MERGE_MESSAGE_SIZE=0

OK. so that will only work for gitea v1.15? Any ETA? 🙃 

Will gitea v1.14.x ignore setting `POPULATE_SQUASH_COMMENT_WITH_COMMIT_MESSAGES=true` ?

**Comment by viceice** [comment]
Fixed by #16134 in gitea v1.15

**Comment by viceice** [comment]
@zeripath Can you add this to the v1.15 milestore?
